### DISCUSSION TITLE
I have an old wallet with deprecation when I sign in

### DISCUSSION THREAD

**Discussion by Donniefresher** [iotaledger/firefly#7758] [Help & Support]
I don’t ever remember having a “password” to send iota? The app asks for one when I try to send I do have my seed phrases please 🙏 let me know how to rectify thanks in advance

**Comment by begonaalvarezd** [comment]
Just in case, please make sure you are using the latest version of the app, the network was recently upgraded. 
If you didnt have ledger to access the wallet, you were asked to provide a password when you setup the wallet for the first time, if you dont remember you can always just create a new profile using your 24 words mnemonic and set and backup the new password for the next time

**Comment by Donniefresher** [comment]
There is no firefly app anymore! Lol noisy deleted it to reinstall it and I
cannot find it on the App Store

**Reply by begonaalvarezd** [reply]
the mobile app was deprecated months ago, you should use the desktop version https://firefly.iota.org/

**Comment by Donniefresher** [comment]
Ok 👍 thank you
